docs/sources/whatsnew/whats-new-next/README.md

```diff
@@ -0,0 +1,57 @@
+# Contribute to 'What's New in Grafana Cloud'
+
+To have a feature presented in [What's New in Grafana Cloud](https://grafana.com/docs/grafana-cloud/whatsnew/), add an entry to the [`index.md`](./index.md) file in this directory.
+
+Use the following template, replace any `<VARIABLE>` with the appropriate text, explained later:
+
+```markdown
+## <FEATURE>
+
+<!-- <CONTRIBUTOR> -->
+<!-- <AVAILABILITY> -->
+
+<CLOUD AVAILABILITY> in Grafana <CLOUD OFFERING>
+
+<DESCRIPTION>
+```
+
+## _`CONTRIBUTOR`_
+
+The name of the contributor of the feature.
+The information is intentionally commented out so that it isn't displayed in the published page.
+
+## _`AVAILABILITY`_
+
+One of:
+
+- OSS
+- Enterprise
+
+Intended availability of the feature when released outside of Grafana Cloud.
+The information is intentionally commented out so that it isn't displayed in the published page.
+
+## _`CLOUD AVAILABILITY`_
+
+One of:
```

Comment:
Since we're doing a whole read-me, might as well point folks to the descriptions.
```suggestion
One of the following [release life cycle stages](/docs/release-life-cycle/):
```

Reply:
I edited your comment to fix the link so I can add it to the batch.